js/scripts/release.sh

```diff
@@ -89,11 +89,13 @@ fi
 
 echo "*** Updating cargo parity-ui-precompiled#$PRECOMPILED_HASH"
 git submodule update
+sed -i "/^parity-ui-precompiled/ { s/branch = \".*\"/branch = \"$BRANCH\"/g; }" dapps/ui/Cargo.toml
```

Comment:
Just out of curiosity : why `"` instead of `'` (which would permit having `branch = "$BRANCH"`) ?